src/v2/controllers/FormController.js

```diff
@@ -132,8 +132,13 @@ export default Controller.extend({
 
   handleInvokeAction(actionPath = '') {
     const idx = this.options.appState.get('idx');
+    let useInteractionCodeFlow;
 
     if (actionPath === 'cancel') {
+      const authClient = this.options.settings.getAuthClient();
+      const transactionMeta = authClient.transactionManager.load();
+      useInteractionCodeFlow = !!transactionMeta?.interactionHandle;
```

Comment:
Why are we using presence of interactionHandle in storage as "source of truth" instead of settings.get('useInteractionFlow')? How could we have transaction storage containing an interactionHandle if our config does not have `useInteractionCodeFlow`?

Because the widget is currently accepting an option, `interactionHandle` which is accepted instead of `useInteractionCodeFlow`:
https://github.com/okta/okta-signin-widget/blob/master/src/v2/client/startLoginFlow.js#L37
https://github.com/okta/okta-signin-widget/blob/master/src/models/Settings.js#L247

There is a task to remove this option, as it has been deemed an anti-pattern: OKTA-405474
Please add a code comment here
```
// TODO: use settings as SoT for `useInteractionCodeFlow` OKTA-405474
```

Reply:
We can't rely on `useInteractionFlow` because it's cleared on this line:
https://github.com/okta/okta-signin-widget/blob/9d9dd22e24a2938548ecfbeb3b69f16a8bfb20fd/src/v2/BaseLoginRouter.js#L219

Reply:
That's odd, I wonder why it is being unset. Will be investigated along with OKTA-405474.

Can you also tag these lines with 
```
// TODO: do not unset  `useInteractionCodeFlow` OKTA-405474
```

Reply:
Added